be/src/exec/pipeline/runtime_filter_types.h

```diff
@@ -315,11 +315,27 @@ class PartialRuntimeFilterMerger {
         }
 
         const auto& num_bloom_filters = _bloom_filter_descriptors.size();
+
+        // remove empty params that generated in two cases:
+        // 1. the corresponding HashJoinProbeOperator is finished in short-circuit style because HashJoinBuildOperator
+        // above this operator has constructed an empty hash table.
+        // 2. the HashJoinBuildOperator is finished in advance because the fragment instance is canceled
+        _partial_bloom_filter_build_params.erase(
```

Comment:
we would better comment why in filter don't have to remove empty params

Reply:
in function merge_local_in_filters, empty vectors of in-filters are aslo removed.